isis/src/base/objs/CameraStatistics/CameraStatistics.cpp

```diff
@@ -441,6 +441,7 @@ namespace Isis {
 
     PvlGroup pAspectRatio("AspectRatio");
     pAspectRatio += constructKeyword("AspectRatioMinimum", m_aspectRatioStat->Minimum());
+    // Note: Maximum is spelled wrong here.
     pAspectRatio += constructKeyword("AspectRatioMaximun", m_aspectRatioStat->Maximum());
```

Comment:
Noticed this misspelling, but didn't want to change it / potentially break existing tests

Reply:
I think this should probably be fixed at some point. If not here, could you put in a ticket?

Reply:
Don't fix this just yet. Put in the issue if it doesn't exist already. We This is parsed by a fair number of scripts so we want to make sure people know it's changing.